etl/storage/storage_test.go

```diff
@@ -710,10 +710,8 @@ func TestStorage_GetLifeline(t *testing.T) {
 
 	objRef := gen.ID()
 
-	genRecords := testutils.OrderedRecords(t, testDB, jetDrop, objRef, 3)
-	testutils.OrderedRecords(t, testDB, jetDrop, gen.ID(), 3)
-
-	expectedRecords := []models.Record{genRecords[2], genRecords[1], genRecords[0]}
+	genRecords := testutils.OrderedStates(t, testDB, jetDrop, objRef, 3)
```

Comment:
Create records (states) for different lifeline here too, as it was. It helps to check that only states for specific object will be return in reply.